test/ibmq/runtime/test_runtime.py

```diff
@@ -239,6 +239,74 @@ def test_jobs_skip_limit(self):
         rjobs = self.runtime.jobs(skip=4, limit=2)
         self.assertEqual(2, len(rjobs))
 
+    def test_jobs_pending(self):
+        """Test retrieving pending jobs (QUEUED, RUNNING)."""
+        jobs = []
+        program_id = self._upload_program()
+        jobs = self._populate_jobs_with_all_statuses(jobs=jobs, program_id=program_id)
+        rjobs = self.runtime.jobs(pending=True)
+        self.assertEqual(5, len(rjobs))
```

Comment:
It'd be nice if the number of pending jobs can be calculated dynamically (e.g. based on the status of each job in `jobs`), instead of hard coding a magic number 5.

Reply:
Fixed!